packages/default/scss/menu-button/_index.scss

```diff
@@ -2,6 +2,18 @@
 @import "../utils/_index.scss";
 
 
+// Module meta
+$_kendo-module-meta: (
+    name: "menu",
```

Comment:
name should be menu-button
change dependencies with the correct ones for the menu-button